.github/workflows/validate.yml

```diff
@@ -131,6 +140,11 @@ jobs:
           fi
           echo "FLAGS=$FLAGS" >> $GITHUB_ENV
 
+      - name: Allow newer dependencies when built with latest GHC
+        if: ${{ matrix.ghc }} == '9.6.1'
+        run: |
+          echo "allow-newer: *:base" >> cabal.project.validate
```

Comment:
what about list all dpeendencies needing a newer base? it is more higienyc and help to keep track of which dependencies needs to be revised (so it is easy to know when you can *remove* it)